docs/metrics/pr-curve.md

```diff
@@ -0,0 +1,173 @@
+| --- | --- | --- | --- | --- |
+| Positive | 0.9 | <span class="mg-cell-color-positive">Positive</span> | <span class="mg-cell-color-positive">Positive</span> | <span class="mg-cell-color-positive">Positive</span> |
+| Positive | 0.7 | <span class="mg-cell-color-positive">Positive</span> | <span class="mg-cell-color-positive">Positive</span> | <span class="mg-cell-color-negative">Negative</span> |
+| Positive | 0.4 | <span class="mg-cell-color-positive">Positive</span> | <span class="mg-cell-color-negative">Negative</span> | <span class="mg-cell-color-negative">Negative</span> |
+| Negative | 0.6 | <span class="mg-cell-color-positive">Positive</span> | <span class="mg-cell-color-positive">Positive</span> | <span class="mg-cell-color-negative">Negative</span> |
+| Negative | 0.4 | <span class="mg-cell-color-positive">Positive</span> | <span class="mg-cell-color-negative">Negative</span> | <span class="mg-cell-color-negative">Negative</span> |
+| Negative | 0.25 | <span class="mg-cell-color-positive">Positive</span> | <span class="mg-cell-color-negative">Negative</span> | <span class="mg-cell-color-negative">Negative</span> |
+| Negative | 0.3 | <span class="mg-cell-color-positive">Positive</span> | <span class="mg-cell-color-negative">Negative</span> | <span class="mg-cell-color-negative">Negative</span> |
+
+</center>
+
+And the precision and recall values for class `Airplane` can be computed:
+<center>
+
+| Threshold | TP | FP | FN | `Airplane` Precision | `Airplane` Recall |
+| --- | --- | --- | --- | --- | --- |
+| **0.25** | 3 | 4 | 0 | $\frac{3}{7}$ | $\frac{3}{3}$ |
+| **0.5** | 2 | 1 | 1 | $\frac{2}{3}$ | $\frac{2}{3}$ |
+| **0.75** | 1 | 0 | 2 | $\frac{1}{1}$ | $\frac{1}{3}$ |
+
+</center>
+
+We are going to repeat this step to compute precision and recall for class `Boat` and `Car`.
+
+<center>
+
+| Threshold | `Airplane` Precision | `Airplane` Recall | `Boat` Precision | `Boat` Recall | `Car` Precision | `Car` <br> Recall |
+| --- | --- | --- | --- | --- | --- | --- |
+| **0.25** | $\frac{3}{7}$ | $\frac{3}{3}$ | $\frac{2}{4}$ | $\frac{2}{2}$ | $\frac{2}{3}$ | $\frac{2}{2}$ |
+| **0.5** | $\frac{2}{3}$ | $\frac{2}{3}$ | $\frac{1}{1}$ | $\frac{1}{2}$ | $\frac{1}{1}$ | $\frac{1}{2}$ |
+| **0.75** | $\frac{1}{1}$ | $\frac{1}{3}$ | $\frac{0}{0}$ | $\frac{0}{2}$ | $\frac{0}{0}$ | $\frac{0}{2}$ |
+
+</center>
+
+Using these precision and recall values, per class PR curves can be plotted:
+
+<center>
+![pr.png](../assets/images/metrics-prcurve-example-multiclass.png)
+</center>
+
+
+## Area Under the PR Curve (AUPRC)
+
+The area under the PR curve (AUPRC) is a **threshold-independent** metric that summarizes the performance of a model
+depicted by a PR curve. The greater the area, the better a model performs. The
+[average precision](./average-precision.md) is one particular method for calculating the AUPRC. With PR curves, we
+don’t have to go into any math and can visually conclude which curves indicate that a certain class or model has a
```

Comment:
nit: `don't have to go into any math` feels extraneous -- `we can visually conclude...` is more pithy.